x-pack/legacy/plugins/graph/public/templates/_graph.scss

```diff
@@ -13,11 +13,15 @@
  * 1. Calculated px values come from the open/closed state of the global nav sidebar
  */
 
+
+ #graphBasic {
+   height: 100%;
```

Comment:
The problem with using 100% height, is that it causes an unnecessary scrollbar for the page

<img src="https://d.pr/free/i/brWPGj+" />

Can you find a solution that doesn't scroll the page? Probably something involving flex.

Reply:
Made another css attempt.
Lemme know what you think.